test/cmdlineTests/yul_function_name_clashes/args

```diff
@@ -1 +1 @@
---strict-assembly --debug-info none
\ No newline at end of file
+--strict-assembly --debug-info none --bin --asm --ewasm --ewasm-ir --ir-optimized
```

Comment:
The "name clashes" tests do not produce any ewasm code so ewasm output is superfluous..
```suggestion
--strict-assembly --debug-info none --bin --asm --ir-optimized
```